Description: Add fox-config man page and fix reswrap man page
Author: Joachim Wiedorn <joodebian at joonet.de>
Forwarded: no
Last-Update: 2016-03-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -urN s05/fox-config-1.6.1 s06/fox-config-1.6.1
--- s05/fox-config-1.6.1	1970-01-01 01:00:00.000000000 +0100
+++ s06/fox-config-1.6.1	2012-05-14 15:14:49.842785156 +0200
@@ -0,0 +1,45 @@
+.TH fox-config 1 "May 2012" "The FOX C++ GUI Toolkit" "FOX Programmers Manual"
+
+.SH NAME
+fox-config \- wrapper around pkg-config
+.SH SYNOPSIS
+.B fox-config
+.RB [ OPTIONS ]
+
+.SH DESCRIPTION
+.B Fox-config
+is a wrapper around pkg-config calls to avoid multilib conflicts.
+
+.SH OPTIONS
+.TP
+\fB\--prefix\fR[\fB=DIR\fR]
+Show the prefix directory.
+.TP
+\fB\--exec-prefix\fR[\fB=DIR\fR]
+Show the exec-prefix directory.
+.TP
+\fB\--version\fR
+Show the libFOX version.
+.TP
+\fB\--libs\fR
+Show all needed libraries for applications using libFOX.
+.TP
+\fB\--cflags\fR
+Show all cflags.
+
+.SH AUTHOR
+This manpage was originally written by Joachim Wiedorn <joodebian@joonet.de>
+for the Debian distribution of the FOX Toolkit.
+
+.SH COPYRIGHT
+Copyright \(co 1997-2016 by Jeroen van der Zijp.
+.br
+This is free software; see the source for copying conditions.  There is
+NO warranty; not even for MERCHANTABILITY or FITNESS FOR  A  PARTICULAR
+PURPOSE.
+.SH SEE ALSO
+The fox-config program is a part of the FOX GUI Toolkit.  Further information
+can be found at:
+
+.B http://www.fox-toolkit.org
+
diff -urN s05/utils/reswrap.1 s06/utils/reswrap.1
--- s05/utils/reswrap.1	2012-04-27 18:38:51.000000000 +0200
+++ s06/utils/reswrap.1	2012-05-14 14:34:51.532865783 +0200
@@ -136,7 +136,7 @@
 Output the size of the resource in the declaration of a resource-array.
 This allows the C++ sizeof() operator to return the correct size of the resource
 even for external declarations.
-Note that in text mode, (-t option), an extra byte is added to the size for the
+Note that in text mode, (\-t option), an extra byte is added to the size for the
 end of string character.
 
 .TP
